en/creating-artboard-adobexd.md

```diff
@@ -0,0 +1,216 @@
+---
+title: Creating An Artboard in Adobe XD
+_description: How to Adobe XD Artboard for a registration screen using Indigo Design Libraries.
+_keywords: Design Systems, Design Systems UX, UI kit, Adobe XD, Ignite UI for Angular, Adobe XD to Angular, Angular, Angular Design System, Export code from Adobe XD, Design Kits for Angular, Adobe XD HTML, Adobe XD to HTML, Adobe XD UI kits
+---
+
+# Creating Your First Artboard in Adobe XD
+
+At the end of this topic you'll have an Adobe XD Artboard for a registration screen using the **Indigo.Design System**. If you use Sketch for design then jump to [this topic](creating-artboard-sketch.md).
+
+<div class="divider--half"></div>
+<img class="responsive-img" src="images/getting-startedxd7.png" srcset="images/getting-startedxd7@2x.png 2x" />
+<div class="divider--half"></div>
+<div class="divider--half"></div>
+<div class="divider--half"></div>
+<div class="divider--half"></div>
+<div class="divider--half"></div>
+
+## What you need
+
+If the [Titillium Web Font][1] isn't installed on your computer, please do it before starting. All texts in the **Indigo.Design System** use it.
+
+## Step by step
+
+### 1. Add the **Indigo.Design System** in Adobe XD.
+
+1.  Download the [**Indigo.Design System**](https://cloud.indigo.design/home).
```

Comment:
Download it from the Getting started menu - in this way you will specify where exactly is the UI kit, since the Getting started menu is not very intuitive.

Reply:
Can you provide me with a URL that shows this menu open?

Reply:
URL cannot be provided, we need to attach a screenshot, because that menu is actually a dropdown only 
<img width="1792" alt="Screen Shot 2021-07-23 at 1 30 56 PM" src="https://user-images.githubusercontent.com/41998471/126770225-d09ce5a7-0575-420c-ae8e-03b47c32426e.png">

Reply:
ОК let's keep things as they are and think on ways to make this more intuitive. Other topics explain about this dropdown, so let's not duplicate this content at this point as we want to improve the overall experience here.